include/libopencm3/stm32/l0/rcc.h

```diff
@@ -457,6 +477,15 @@
 #define RCC_CSR_LSIRDY				(1 << 1)
 #define RCC_CSR_LSION				(1 << 0)
 
+/* --- Helper Defines ----------------------------------------------------- */
+/* MSI Base frequency is 4194304 Hz can be divided by 2, 4, 8, 16, 32, 64 */
+#define RCC_MSI_4MHZ				((0x6 << 24) | 4194304)
+#define RCC_MSI_2MHZ				((0x5 << 24) | 2097152)
+#define RCC_MSI_1MHZ				((0x4 << 24) | 1048576)
+#define RCC_MSI_524KHZ				((0x3 << 24) | 524288)
+#define RCC_MSI_262KHZ				((0x2 << 24) | 262144)
+#define RCC_MSI_131KHZ				((0x1 << 24) | 131072)
+#define RCC_MSI_65KHZ				((0x0 << 24) | 65536)
```

Comment:
No, just the available frequencies from the data sheet, I store the config bits in the upper 8 bits of the `#define` and the actual frequency in the lower 24 bits. This lets it provide both the configuration data and the frequency that will be used (which is unlike the HSI which is fixed frequency).